code/game/mecha/equipment/mecha_equipment.dm

```diff
@@ -130,6 +131,8 @@
 	return
 
 /obj/item/mecha_parts/mecha_equipment/proc/detach(atom/moveto=null)
+	if(!detachable)

```

Comment:
Don't these get detached during destroy and similar, this should probably only apply on manual removal.

Reply:
Far as I can tell, the destroy() proc doesn't actually call detach, and does all the work manually instead. That being said, this check probably isn't necessary since I removed the actual detach button (well, link) from equipment with detachable = FALSE anyway. I'll remove this.